### DISCUSSION TITLE
First day Monday

### DISCUSSION THREAD

**Discussion by MarSalLT** [dmuy/duDatepicker#54] [Q&A] [answered]
Hi, how to make this calendar to start on Monday not Sunday?

**Comment by MarSalLT** [comment]
@dmuy Any suggestions?

**Reply by dmuy** [reply, marked as answer]
Hi @Xmenas,

In the [configurations wiki](https://github.com/dmuy/duDatepicker/wiki/Options#configurations), there's an option `firstDay`, you can set it from `1-7` which is `Monday-Sunday`.
